PrismaDB-QueryAST/DML/Expressions/BooleanExpressions.cs

```diff
@@ -37,10 +37,11 @@ public class BooleanLike : BooleanExpression
     {
         public ColumnRef Column;
         public StringConstant SearchValue;
+        public StringConstant EscapeChar;
```

Comment:
@cheziyi can you check if this has to be `StringConstant` and not `string`? I think we use `string` in other places like that.

Reply:
Yes, it needs not be a StringConstant. Technically it can be a nullable char, do not need to be a string.

Reply:
And the `SearchValue`?

Reply:
`SearchValue` should remain as a `StringConstant`, as it follows the rules of a normal SQL string literal, and will be parsed as such in the parsers.

Reply:
Ok